invenio_rdm_records/fixtures/data/vocabularies/roles.yaml

```diff
@@ -0,0 +1,63 @@
+- id: contactperson
+  title:
+    en: Contact Person
```

Comment:
You'll need to check the DataCite export format - we likely need to add the mapping to the datacite role

Reply:
https://github.com/inveniosoftware/invenio-rdm-records/issues/699